README.md

```diff
@@ -43,3 +43,10 @@ Mapgl-ruler uses github-flow to accept & merge fixes and improvements. Basic pro
 - Create a new pull request to original repo.
 
 Pull requests with failing tests will not be accepted. Also, if you add or modify packages to `package.json`, make sure you use `npm` and update `package-lock.lock`.
+
+## Tests
+
+For check or update screenshots use command
+```shell
+docker-compose up --build
```

Comment:
а она же только контейнер запустит, чо дальше делать: запускать тесты или обновлять скрины - надо самому сообразить и внутри контейнера выполнить? может тогда надо это более явно написать или дописать команду тут? или я ошибаюсь?

Reply:
Эта команда запускает контейнер, а в самом контейнере уже прописана команда `npm test`, которая запускает все тесты.

Но да, команды на обновление тестов не хватает.

Reply:
Теперь тут надо написать про новые команды. Еще кажется нужно дать небольшой пояснение о том, почему мы гоняем тесты в докере. Типа (у меня правда может быть кривой инглиш):

To avoid font rendering issues on different platforms, screenshot tests running via docker.